stl/inc/random

```diff
@@ -2025,6 +2027,39 @@ basic_ostream<_Elem, _Traits>& operator<<(basic_ostream<_Elem, _Traits>& _Ostr,
     return _Dist._Write(_Ostr);
 }
 
+// FUNCTION TEMPLATE _Float_upper_bound
+
+// Returns smallest _Flt s.t. static_cast<_Ty>(_Result) > _Val. First
+// truncate to largest _Flt <= _Val, then add ceil(ulp)
+
+template <class _Flt, class _Ty, typename = enable_if_t<is_unsigned_v<_Ty> && is_floating_point_v<_Flt>>>

```

Comment:
1. Because this is an internal function, do we need a SFINAE constraint? SFINAE is important when we're dealing with an overload set, or when we want type traits like `is_constructible` to report useful answers. Here, if we just want to enforce that the internal function is called correctly, `static_assert` within the function body is simpler and will result in better error messages for misuse.
2. (moot point due to (1)) We conventionally use `enable_if_t<SomeCondition, int> = 0` for SFINAE in product code, which works in more situations.
3. (moot point due to (1)) We conventionally say `class` in product code instead of `typename` (although I would like to globally change to `typename` someday, that would be a lot of syntax changes for little value).